### PR TITLE
Fix Runcam Split command box names and move to localized resource file

### LINKED ISSUE #649: "Camera control" Modes displaying old/wrong text first time
Runcam Split configured on UART2.
Connecting to current 3.2.0 (as of today) using the latest (as of today) configurator and clicking on modes results in this screen:
2017-09-02 23_37_51-betaflight - configurator
If I then click on another tab (i.e. receiver or configuration) and back on the modes tab, the names change to the correct ones:
2017-09-02 23_38_16-betaflight - configurator
As soon as I disconnect and reconnect, it returns to showing the wrong names again until I change tabs.

### PR DESCRIPTION
**Pull request by basdelfos**
Fixes #649

### CODE REVIEW

File: tabs/auxiliary.js [hunk 1]
```diff
@@ -36,28 +36,16 @@ TABS.auxiliary.initialize = function (callback) {
...
-            case 32: // BOXCAMERA1
-                return "CAMERA WI-FI";
-            case 33: // BOXCAMERA2
-                return "CAMERA POWER";
-            case 34: // BOXCAMERA3
-                return "CAMERA CHANGE MODE"
-            default:
-                return originalModeName;
-        }
+    function localizeBoxNameWithModeID(modeId, originalModeName) {
```
Comment (mikeller) [inline tabs/auxiliary.js]: This is wrong. These switches were defined for generic use for cameras in the firmware for a reason. If this behaviour is not wanted any more, the switches should be renamed to be specific to RunCam in the firmware.

I think the original intent to only override the labels if a port for a RunCam is configured is correct, and if the labels do not get propery updated in this case this (and only this) should be fixed.

Comment (basdelfos) [reply tabs/auxiliary.js]: Aha, I didn't knew that these camera switches where intended to be used for generic camera controls, I thought there were specific to the RunCam Split. That figures why the code was setup like it was. I will revert this back and fix the update issue, but I will leave in the localized part (have to change it a little bit).

The update issue is probably the case that the ports config is not yet loaded from the fc. Port config is only loaded from the fc in specific tabs, that is why that after opening another tab, the labels are displayed correctly.

Comment (mikeller) [reply tabs/auxiliary.js]: Yes, adding the messages to localisation is definitely a good and wanted change. And I think you are right, the port configuration not having been loaded is most likely the culprit. There normally is a string of MSP commands in the js for every tab that are sent whenever the tab is opened, and it should be sufficient to copy / paste the one MSP command that loads the ports config from the ports tab into the modes tab.

**Comment by mikeller** [comment]
Looking good now. Will test tonight.

**Pull request merged**